### ISSUE TITLE
problem with type="normal" in 5.1.0?

### ISSUE THREAD

**Issue by yuomtheara** [aldeed/meteor-autoform#832]
I have like this:
```
WelcomeConfig = new SimpleSchema({
    name: {
        type: String,
        label: 'Module'
     }
});
---------------------------------------------------
{{#autoForm schema=WelcomeConfig id="welcomeConfig" type="normal"}}
        {{> afQuickField name="name"}}
        <button type="submit" class="btn btn-primary">Go ...</button>
    {{/autoForm}}
---------------------------------------------------
AutoForm.hooks({
    welcomeConfig: {
        onSubmit: function (insertDoc, updateDoc, currentDoc) {
            this.event.preventDefault();
            Router.go('home.index');
            this.done();
        }
    }
});

```
I always get the `required error`, event if i fill the value.

**Comment by gbaladi** [comment]
same here

**Comment by djhi** [comment]
If you add an onError callback, you'll see that's the same prb than #829

**Comment by aldeed** [comment]
@yuomtheara @gbaladi is this fixed by 5.1.1?

**Comment by yuomtheara** [comment]
Thanks, I will try again.
